src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,20 +11,23 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
```

Comment:
- This commit is unrelated to the concerned tutorial exercise.
- Avoid accumulating unrelated changes into a single PR. 
- Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.